javaparser-core/src/main/java/com/github/javaparser/printer/lexicalpreservation/changes/Change.java

```diff
@@ -41,6 +42,10 @@ default boolean evaluate(CsmConditional csmConditional, Node node) {
             case IS_PRESENT:
                 return !Utils.valueIsNullOrEmptyStringOrOptional(getValue(csmConditional.getProperty(), node))
                 		&& !isEvaluatedOnDerivedProperty(csmConditional.getProperty());
+	        case IS_STMT_GROUP_ENTRY:
```

Comment:
I don't understand this but it must be linked to other changes?

Reply:
I don't think this should be necessary if it's possible to use the FLAG condition.